cmd/init.go

```diff
@@ -18,14 +18,19 @@ func init() {
 var initCmd = &cobra.Command{
 	Use:   "init [project]",
 	Short: "Initialize a flutter project to use go-flutter",
-	Args: func(cmd *cobra.Command, args []string) error {
-		if len(args) != 1 {
-			return errors.New("requires one argument, the project path. e.g.: github.com/my-organization/my-app")
-		}
-		return nil
-	},
 	Run: func(cmd *cobra.Command, args []string) {
-		projectPath := args[0]
+		var projectPath string
+		if len(args) == 0 || args[0] == "." {
+			projectName := getPubSpec().Name
+			u, err := user.Current()
+			if err != nil {
+				fmt.Printf("hover: Couldn't get current user: %v\n", err)
+			}
+			projectPath = "github.com/" + u.Username + "/" + projectName
```

Comment:
We cannot asume github nor username being an actual user on github. Instead just use projectName. That is fine as long as the project is an app.

((The projectPath as argument must become mandatory when --template=plugin is used (future hover heature)))

Reply:
How about having a `hover init-plugin` command to keep consistency with packaging.

Nevertheless I'll change from github.com... to JUST the projectName